etcdctl/ctlv2/command/util.go

```diff
@@ -156,7 +156,12 @@ func getTransport(c *cli.Context) (*http.Transport, error) {
 		CertFile: certfile,
 		KeyFile:  keyfile,
 	}
-	return transport.NewTransport(tls, defaultDialTimeout)
+
+	to := c.GlobalDuration("total-timeout")
```

Comment:
if totalTimeout !=0 && to < defaultDialTimeout {
    dialTimeout = totalTimeout
}

/cc @gyuho